panther.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Vikas(Vikas_2_0)",
+			"oem": "Google",
+			"device": "Pixel 7",
+			"filename": "crDroidAndroid-15.0-20241106-panther-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/panther/11.x/crDroidAndroid-15.0-20241106-panther-v11.0-BETA.zip/download",
+			"timestamp": 1730880151,
+			"md5": "4a26b29d392005fa9e256e6ec11c0752",
+			"sha256": "74187369cc84ff1656aff5725233fbe99602419ab735ddc4e41dac0fb0f8c955",
+			"size": 1581910983,
+			"version": "11.0",
+			"buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/rom-official-panther-15-0-crdroidandroid-v11-x.4702074/",
```

Comment:
as has been discussed few times already, please update forum post as per latest template
footer is bad